libs/core/src/lib/dialog/dialog-footer/dialog-footer.component.ts

```diff
@@ -22,7 +22,9 @@ export class DialogFooterComponent implements AfterContentInit {
     /** @hidden */
     @ContentChildren(TemplateDirective) customTemplates: QueryList<TemplateDirective>;
 
-    constructor(@Optional() @Inject(DIALOG_CONFIG) public dialogConfig: DialogConfig) {}
+    constructor(@Optional() @Inject(DIALOG_CONFIG) public dialogConfig: DialogConfig) {
```

Comment:
do we need the `public` here?

Reply:
Yes, this is a shortcut for creating object prop, otherwise this will be a normal constructor parameter.